catalog/CatalogTypedefs.hpp

```diff
@@ -36,6 +36,10 @@ typedef int attribute_id;
 typedef std::size_t partition_id;
 typedef int numa_node_id;
 
+// Catalog ids use negative values as invalid ids. Mark -1 as constant invalid
+// id for the catalog ids.
+constexpr int kInvalidCatalogId = -1;
```

Comment:
1. No, but you need to move your comment along with `kInvalidCatalogId`, below the comment I mentioned above.
2. Yes, do update the copyright for every file you touch.